i18n/locales/ja_JP.json

```
@@ -10,5 +10,10 @@
     "wip": "準備中...",
     "gettingStartedButton": "はじめる",
     "discoverButton": "サーバーを探す"
+  },
+  "sponsor": {
+    "title": "寄付のお願い",
+    "description": "Pulsate Project は営利目的ではありません。ソフトウェアの開発資金は全て皆さんの寄付などに頼っています。今度もプロジェクトを続けれるように支援をお願いします。",
```

ら抜き言葉になってるので直した方がフォーマルになりそう
```suggestion
    "description": "Pulsate Project は営利目的ではありません。ソフトウェアの開発資金は全て皆さんの寄付などに頼っています。今度もプロジェクトを続けられるように支援をお願いします。",
```

---

25b9f78 で修正しました.